bot.go

```diff
@@ -426,32 +421,20 @@ func issuesHandler(w http.ResponseWriter, r *http.Request) {
 	githubclient := github.NewClient(&http.Client{Transport: &transport})
 
 	lcBody := strings.ToLower(*payload.Issue.Body)
-	lcTitle := strings.ToLower(*payload.Issue.Title)
-
-	if *payload.Action == "opened" {
```

Comment:
Removed the `if` as that is checked here:

https://github.com/i3/i3-github-bot/blob/027111a94b3250ac01744319c69afd0c49831de5/bot.go#L412-L414